Updated docs for PG 17 support (#60)
percona/pgsm-docs: docs/reference.md

     | local_blks_written | bigint                      | Total number of local blocks  written simultaneously to a drive during the query execution
     | temp_blks_read     | bigint                      | Total number of blocks of temporary files read from a drive. Temporary files are used when there's not enough memory to execute a query
     | temp_blks_written  | bigint                      | Total number of blocks of temporary files written to a drive

>      | shared_blk_read_time      | double precision            | Total waiting time (in ms) for reading shared blocks
>      | shared_blk_write_time     | double precision            | Total waiting time (in ms) for writing shared blocks to a drive
>      | local_blk_read_time       | double precision  | Total time the statement spent on reading local blocks, in ms
>      | local_blk_write_time      | double precision  | Total time the statement spent on writing local blocks, in ms

Reviewer: ```suggestion
     | shared_blk_read_time      | double precision            | Total time the statement spent reading shared blocks, in milliseconds
     | shared_blk_write_time     | double precision            | Total time the statement spent writing shared blocks, in milliseconds
     | local_blk_read_time       | double precision  | Total time the statement spent on reading local blocks, in milliseconds
     | local_blk_write_time      | double precision  | Total time the statement spent on writing local blocks, in milliseconds
```

Let's use descriptions from `pg_stat_statements` docs

>      | temp_blk_read_time | double precision            | Total number of temp blocks read by the statement 
>      | temp_blk_write_time| double precision            | Total number of temp blocks written by the statement

Reviewer: Fix descriptions
```suggestion
     | temp_blk_read_time | double precision            | Total time the statement spent reading temporary file blocks, in milliseconds 
     | temp_blk_write_time| double precision            | Total time the statement spent writing temporary file blocks, in milliseconds
```

     | jit_optimization_time| double precision     | Total time spent by the statement on optimizing, in milliseconds 
     | jit_emission_count   | bigint               | Number of times code has been emitted
     | jit_emission_time    | double precision     | Total time spent by the statement on emitting code, in milliseconds
     | jit_deform_count     | bigint               | Total number of tuple deform functions JIT-compiled by the statement | 
     jit_deform_time       | double precision | Total time spent by the statement on JIT-compiling tuple deform functions, in milliseconds | 
     | stats_since         | timestamp with time zone| Time at which statistics gathering started for this statement|

>      | minmax_stats_since  | timestamp with time zone| Time at which min/max statistics gathering started for this statement (fields min_plan_time, max_plan_time, min_exec_time and max_exec_time) |

Reviewer: ```suggestion
     | minmax_stats_since  | timestamp with time zone| Time at which min/max statistics gathering started for this statement (fields min_plan_time, max_plan_time, min_exec_time and max_exec_time). Please note that unlike `pg_stat_statements` `pg_stat_monitor` doesn't have option to reset min/max counters, so `minmax_stats_since` always equals to `stats_since` |
```

     | shared_blks_written | bigint                     | Shows the number of shared memory blocks written simultaneously to a drive during the query execution
     | local_blks_hit     | bigint                      | The number of blocks which are considered as local by the backend and thus are used for temporary tables
     | local_blks_read    | bigint                      | Total number of local blocks read during the query execution
     | local_blks_dirtied | bigint                      | Total number of local blocks "dirtied" during the query execution (i.e. a query modified at least one tuple in a block and this block must be written to a drive)
     | local_blks_written | bigint                      | Total number of local blocks  written simultaneously to a drive during the query execution
     | temp_blks_read     | bigint                      | Total number of blocks of temporary files read from a drive. Temporary files are used when there's not enough memory to execute a query
     | temp_blks_written  | bigint                      | Total number of blocks of temporary files written to a drive

>      | blk_read_time      | double precision            | Total waiting time (in ms) for reading blocks
>      | blk_write_time     | double precision            | Total waiting time (in ms) for writing blocks to a drive

Reviewer: ```suggestion
     | blk_read_time      | double precision            | Total time the statement spent reading data file blocks, in milliseconds
     | blk_write_time     | double precision            | Total time the statement spent writing data file blocks, in milliseconds
```

>      | temp_blk_read_time | double precision            | Total number of temp blocks read by the statement 
>      | temp_blk_write_time| double precision            | Total number of temp blocks written by the statement

Reviewer: ```suggestion
     | temp_blk_read_time | double precision            | Total time the statement spent reading temporary file blocks, in milliseconds
     | temp_blk_write_time| double precision            | Total time the statement spent writing temporary file blocks, in milliseconds
```
